source/adapters/level_zero/v2/queue_immediate_in_order.cpp

```diff
@@ -1080,14 +1081,17 @@ ur_result_t ur_queue_immediate_in_order_t::enqueueGenericCommandListsExp(
   return UR_RESULT_SUCCESS;
 }
 
-ur_result_t ur_queue_immediate_in_order_t::enqueueCommandBuffer(
-    ze_command_list_handle_t commandBufferCommandList,
-    ur_event_handle_t *phEvent, uint32_t numEventsInWaitList,
-    const ur_event_handle_t *phEventWaitList) {
+ur_result_t ur_queue_immediate_in_order_t::enqueueCommandBufferExp(
+    ur_exp_command_buffer_handle_t hCommandBuffer, uint32_t numEventsInWaitList,
+    const ur_event_handle_t *phEventWaitList, ur_event_handle_t *phEvent) {
+
+  ze_command_list_handle_t commandBufferCommandList =
+      hCommandBuffer->commandListManager.getZeCommandList();
```

Comment:
@pbalcer Since you approved this PR there have been git conflicts with the v2 merge I've had to resolve, can you re-review?

Reply:
lgtm. thanks.